sct@skulptist.de/files/sct@skulptist.de/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+    "uuid": "sct@skulptist.de",
+    "author": "Matthias Hermsdorf",
+    "version": "1.0.0",
+    "name": "SCT Toggle",
+    "description": "Toggle screen temperatures with sct. \nSet up to 7 custom color temperatures and toggle through by clicking on the applet",
+    "max-instances": "1"
```

Comment:
Can this support more than one instance for multi-monitor setups?

Reply:
sct has no option to set the screen color temperature for different screens. So I have had set the max-instances to 1.
I will try what will happen with more instances.

Reply:
At the moment every applet would have its own state. Think about having 7 steps configurated and click on one applet to the 5. and then clicking on another instance, it would switch from 1 to 2. 
With more instances they should share the currentStep state. Yes, its implementable.